src/imars3d/ui/widgets/hyperstack_view.py

```diff
@@ -0,0 +1,198 @@
+#!/usr/bin/env python3
+"""Hyperstack viewer for iMars3D.
+
+To test the hyperstack viewer widget in a Jupyter notebook:
+
+import panel as pn
+from imars3d.ui.widgets.hyperstack_view import HyperstackView
+
+pn.extension()
+hv.extension("bokeh")
+
+viewer = HyperstackView(data=skimage.data.brain())
+
+now you can use either
+
+- viewer or pn.Row(viewer) or viewer.servable() to check the widget in cell
+- viewer.show() to check the widget in a separate window (on a bokeh server)
+"""
+import numpy as np
+import panel as pn
+import param
+import holoviews as hv
+from holoviews import opts
+from holoviews import streams
+from imars3d.ui.widgets.viewer2d import Viewer2D
+
+logger = param.get_logger(__name__)
```

Comment:
So, these are getting in a separate pr right?

Reply:
:sweat: good catch, I will fix it now.

Reply:
all the loggers in the backend are fixed in a separate PR, but we should start using `logging.getLogger(__name__)` from now on.